test/fixtures/basic/package.json

```diff
@@ -5,7 +5,6 @@
     "run-series": "^1.1.1"
   },
   "devDependencies": {
-    "ncp": "^2.0.0",
```

Comment:
I added this particular dependency to test #186. If you remove it, please replace it with a module (with zero dependencies) that creates a file in `node_modules/.bin`.